src/engine/MarabouMain.cpp

```diff
@@ -128,7 +128,7 @@ int marabouMain( int argc, char **argv )
     }
     catch ( const Error &e )
     {
-        printf( "Caught a %s error. Code: %u, Errno: %i, Message: %s.\n",
+        fprintf(stderr, "Caught a %s error. Code: %u, Errno: %i, Message: %s.\n",
```

Comment:
frptinf(stderr -> fprintf( stderr

This is the coding convention of files in src.

Reply:
Fixed :+1: